components/feature/toolbar/src/main/java/mozilla/components/feature/toolbar/ToolbarPresenter.kt

```diff
@@ -86,9 +91,28 @@ class ToolbarPresenter(
     override fun onSecurityChanged(session: Session, securityInfo: Session.SecurityInfo) =
         updateToolbarSecurity(securityInfo)
 
+    override fun onTrackerBlockingEnabledChanged(session: Session, blockingEnabled: Boolean) {
+        updateToolbarTrackingProtection(session)
+    }
+
+    override fun onTrackerBlocked(session: Session, tracker: Tracker, all: List<Tracker>) {
+        updateToolbarTrackingProtection(session)
+    }
+
     private fun updateToolbarSecurity(securityInfo: Session.SecurityInfo) =
         when (securityInfo.secure) {
             true -> toolbar.siteSecure = Toolbar.SiteSecurity.SECURE
             false -> toolbar.siteSecure = Toolbar.SiteSecurity.INSECURE
         }
+
+    private fun updateToolbarTrackingProtection(session: Session) {
+        val areTrackersBlocked = session.trackersBlocked.isNotEmpty()
+        val siteTC = when (session.trackerBlockingEnabled) {
```

Comment:
naming nit: What is the `TC` in `siteTC`? 🤔

Reply:
🙃  I wanted to write `siteTP` XD

Reply:
Fixed!